print-ahcp.c

```diff
@@ -102,18 +102,14 @@ ahcp_time_print(netdissect_options *ndo,
                 const u_char *cp, uint8_t len)
 {
 	time_t t;
-	struct tm *tm;
 	char buf[BUFSIZE];
```

Comment:
BUFSIZE is defined in addrtoname.h. It should not be used for this usage.

Reply:
Fixed.